lib/cocoapods/generator/header.rb

```diff
@@ -84,6 +84,7 @@ def save_as(path)
       def generate_platform_import_header
         case platform.name
         when :ios then "#import <UIKit/UIKit.h>\n"
+        when :tvos then "#import <UIKit/UIKit.h>\n"
```

Comment:
I agree, but don't think this PR is the right place to change this stuff. Users will be very confused when UIKit symbols are suddenly undefined for just one of the UIKit platforms.